src/app/auth/login.ts

```diff
@@ -228,3 +380,16 @@ export const isCodeValid = async ({
 
   return TwilioClient().validateVerify({ to: phone, code })
 }
+
+export const verifyJwt = async (token: string) => {
+  const newToken = await sendOathkeeperRequest(token as SessionToken)
+  if (newToken instanceof Error) return newToken
+  const keyJwks = await jwksRsa(getJwksArgs()).getSigningKey()
+  const verifiedToken = jsonwebtoken.verify(newToken, keyJwks.getPublicKey(), {
+    algorithms: ["RS256"],
+  })
+  if (typeof verifiedToken === "string") {
+    throw new Error("tokenPayload should be an object")
```

Comment:
A more precise error should be defined that should be a subclass of the `Error` type, so we could filter it out at the graphql level and provide proper alerting